CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - Coarse horizon cut for reduced computational load.
 - Selection keys in obsparam files (valid options for UVData.select)
 - New batch scripts for profiling jobs on a SLURM-managed cluster, and plotting the results.
+- Checking the beam kernel width makes sense for Airy beams
```

Comment:
I'm a little worried that this won't be in the right place if we merge this as is. I think it needs to be rebased and then check that this change is showing up under unreleased, rather than under version 0.2.1

Reply:
Now that you've rebased, I can tell it's in the wrong place. It needs to be under "unreleased"

Reply:
Missed that... should be fixed now.